README.rst

```diff
@@ -869,23 +869,21 @@ A special thank you
 
 SAM CLI uses the open source
 `docker-lambda <https://github.com/lambci/docker-lambda>`__ Docker
-images created by [@mhart](https://github.com/mhart).
+images created by `@mhart <https://github.com/mhart>`__.
 
 Examples
 --------
 
-You can find sample functions code and a SAM template used in this
-README under the
-`samples <https://github.com/awslabs/aws-sam-local/tree/master/samples>`__
-folder within this repo.
+You can find sample code and SAM templates in this
```

Comment:
Thanks for cleaning this up, @default50!

In "You can find sample code and SAM templates in this", please change this => the. The rest looks good to me.